src/super_gradients/recipes/imagenet_regnetY.yaml

```diff
@@ -1,14 +1,26 @@
-#  RegnetY800 Imagenet classification training:
-#  This example trains with batch_size = 128
-#  Training time on a single GeForce RTX 2080 Ti is ~ 44min / epoch, total time ~ 2d 22h 50m.
-#  Reach => 76.05 (Final averaged model)
-#  Log and tensorboard at s3://deci-pretrained-models/regnetY800-imagenet-tensorboard/
+#  RegnetY Imagenet classification training:
+#  This example trains with batch_size = 256
+#  Training time on a single GeForce RTX 2080 Ti, and top1 accuracies:
+#  11 days for RegnetY200, 70.88
+#  12 days for RegnetY400, 74.74
+#  19 days for RegnetY600, 76.18
+#  20 days for RegnetY800, 77.07
+#  NOTE: Training should probably be lower as resources were shared among the above runs.
+
+#  Logs and tensorboards at:
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY800/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY600/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY400/
+# https://deci-pretrained-models.s3.amazonaws.com/RegnetY200/
 
 # Instructions:
 # Set the PYTHONPATH environment variable: (Replace "YOUR_LOCAL_PATH" with the path to the downloaded repo):
-#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/
+#   export PYTHONPATH="YOUR_LOCAL_PATH"/super_gradients/:"YOUR_LOCAL_PATH"/super_gradients/src/
```

Comment:
now that you moved recipes into src you dont need `"YOUR_LOCAL_PATH"/super_gradients/` anymore